upload/ValidateInterface.php

```diff
@@ -1,11 +1,12 @@
 <?php
+
 namespace upload;
 
 interface ValidateInterface
 {
-  public function Size($AllowedSize);
-  public function Extension($AllowedExtension);
-  public function Exist();
-}
+    public function Size($AllowedSize);
```

Comment:
__[code_sniffer]__
* Method name "ValidateInterface::Size" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->